src/features/architect/utils/contractHistory/contractEventSerialization.ts

```
@@ -0,0 +1,173 @@
+  type ContractLedgerValidation,
+} from './contractEventRecords';
+
+/** Bump only for a format change that older payloads cannot satisfy. */
+export const CONTRACT_EVENT_LEDGER_PAYLOAD_VERSION = 1;
+
+export interface ContractEventLedgerPayload {
+  readonly payloadVersion: number;
+  readonly ledgerId: string;
+  readonly ledgerVersion: number;
+  readonly events: readonly ContractEventRecord[];
+}
```

ESLint

[error] Do not declare Player*/Contract* interfaces outside src/schemas. Import from src/schemas/players_v2 instead.

(no-restricted-syntax)